### PR TITLE
[IMP] base: improve logger message

### PR DESCRIPTION
**Pull request by niyasraphy**
before this commit, if the translation import is failed, in the log it shows "unsuccessfully imported"

after this commit, the logger message is improved and show file import failed

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by VincentSchippefilt** [comment]
Hello, thank you for this fix. 
Could you change the commit message to "[FIX] base: improve logger message" and target 14.0 ?
Also, you modified message contains 2 times "file". IMHO it doesn't real well. 
Maybe we can reword it to `Could not import the file either due to a format mismatch or it is malformed.`

**Comment by niyasraphy** [comment]
@VincentSchippefilt thanks for the feedback,
created new pr targeting the master with suggested changes:   https://github.com/odoo/odoo/pull/113307
